website/blog/2022-07-04-aluminium.md

```diff
@@ -0,0 +1,467 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.7 - Aluminium
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v0.11.7, which brings a large
+number of Scala 3 improvements as well as a few interesting code actions.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">164</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">77</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">9</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/51?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the [Scala Center](https://scala.epfl.ch/)
+and [VirtusLab](https://virtuslab.com) with the help from
+[Lunatech](https://lunatech.com) along with contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Release notes now shown in VS Code
+- Improvements to Bloop Java home automation.
+- Allow to run all scalafix rules on a file
+- [Scala 3] Implement-all abstract members code action.
+- [Scala 3] Add snippet completions.
+- [Scala 3] Add file name completions.
+- [Scala 3] Expression evaluation for debugger.
+- [Scala 2] Add ConvertToNamedArguments code action.
+- New flatMap to for comprehension code action.
+
+## Release notes now shown in VS Code
+
+If you are using Visual Studio Code you should see this message right after the
+update of the Metals extension has finished. 🚀
+
+Starting with the current release we will be showing release notes inside VS
+Code when the version update completes. This way users will know whenever the
+extension was updated and it will not just silently update in the background.
+
+Release notes will be shown only once after the update, but can be viewed again
+by executing `Metals: Show release notes`
+
+This was contributed by [kpodsiad](https://github.com/kpodsiad), thanks for the
+continuous contributions!
+
+## Improvements to Bloop Java home automation
+
+From the
+[last version of Metals](https://scalameta.org/metals/blog/2022/06/03/aluminium#automatically-setup-java-home-for-the-bloop-build-server)
+we introduced an automated setup of Java home for your Bloop build server, which
+is used to compile your code under the hood and is the default build server to
+be used.
+
+Your Java Home would be added when starting Metals, before starting the build
+server, and would also suggest an update when changing the versions in the
+Metals settings. However, this caused a couple of issues:
```

Comment:
IMO it would be nice to include some screenshots to clarify about which notifications we're talking about

Reply:
Added, let me know if it's ok.